src/contexts/LoginContext.tsx

```diff
@@ -44,7 +70,7 @@ const LoginProvider = ({ children }) => {
     removeSites()
     // NOTE: This is REQUIRED (emphasis here) for auto-redirect on removal of stored user id.
     // This is IN ADDITION to removing the value associated with the key.
-    setStoredUserId(null)
+    setStoredUserId("")
```

Comment:
this might be worth double checking against e2e - previously when i tried to make a minor change to this line, it broke (and hence the long comment above) and e2e was failing so we might wanna double check if this is safe

Reply:
Ran a few test suites with no issue, so this should be fine!